test/unit/routes/test-workload-points.js

```diff
@@ -87,6 +89,10 @@ describe('Admin Workload Points route', function () {
     return supertest(app).get(WORKLOAD_POINTS_URL).expect(200)
   })
 
+  it('should respond with 200 when the correct admin/workload-points/t2s url is called', function () {
+    return supertest(app).get(WORKLOAD_POINTS_T2A_URL).expect(200)
+  })
```

Comment:
Can you add to these tests to check that T2A url calls the service with `t2a=true` and regular calls the service with `t2a=false`